scipy/stats/_stats_py.py

```diff
@@ -7501,48 +7501,86 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            in the case of a correlated system of variables is such that it can be reasonably
            supposed to have arisen from random sampling", Philosophical Magazine. Series 5. 50
            (1900), pp. 157-175.
+    .. [4] Mannan, R. William and E. Charles. Meslow. “Bird populations and
+           vegetation characteristics in managed and old-growth forests,
+           northeastern Oregon.” Journal of Wildlife Management
+           48, 1219-1238, :doi:`10.2307/3801783`, 1984.
 
     Examples
     --------
-    When just `f_obs` is given, it is assumed that the expected frequencies
-    are uniform and given by the mean of the observed frequencies.
+    In [4]_, bird foraging behavior was investigated in an old-growth forest
+    of Oregon.
+    In the forest, 44% of the canopy volume was Douglas fir,
+    24% was ponderosa pine, 29% was grand fir, and 3% was western larch.
+    The authors observed the behavior of several species of birds, one of
+    which was the red-breasted nuthatch. They made 189 observations of
+    foraging of this species noting that 23% of birds favoured Douglas
+    fir, 27% ponderosa pine, 29% grand fir, and 21% western larch.
```

Comment:
This comes from:
![image](https://user-images.githubusercontent.com/6570539/210443609-8cf08ebe-2881-4f3f-a161-676d89324522.png)

Unless there's a section in the text that I'm missing, I wouldn't interpret this as meaning that 23% of birds favored Douglas fir. That would correspond with observing 189 different birds and somehow assessing which tree each individual "preferred" (e.g. an individual foraged exclusively in this species of tree).

Instead, I think it means that they recorded 189 observations, and I think each "observation" is a summary of what a bird was doing for a period of 15 seconds.
![image](https://user-images.githubusercontent.com/6570539/210444810-c9cd76e9-8958-47e3-bffe-f8c0cb8e8dc1.png)

I think that in 23% of observations, the bird was foraging in a Douglas fir tree.

As you note, the percentages don't add up to 100%, but we know that there were actually 189 discrete observations, and we can figure out what integer number of observations in each tree would produce the (rounded) percentages reported in the table and sum to 189. I think they are unique.

```suggestion
    which was the red-breasted nuthatch. They made 189 observations of this
    species foraging, recording 43 ("23% ") of observations in Douglas fir, 
    52 ("28%") in ponderosa pine, 54 ("29%") in grand fir, and 40 ("21%") in
    western larch.
```

I think that `f_obs` below should be reported using these integers because I think an "observed frequency" should be integral.